project/Dependencies.scala

```diff
@@ -46,7 +46,9 @@ object Dependencies {
   val scalaNative03Version = "0.3.9"
   val scalaNative04Version = "0.4.0-M2"
   val scalaJs06Version = "0.6.32"
-  val scalaJs10Version = "1.0.1"
+  val scalaJs1Version = "1.3.1"
+  val scalaJsEnvsVersion = "1.1.1"
+  val scalaJsEnvJsdomNodejsVersion = "1.1.1"
```

Comment:
This is unused. (And if were used, it should be 1.1.0.)

Reply:
@lolgab could you remove `scalaJsEnvJsdomNodejsVersion` ?

Reply:
Ops, I was adding it and then I went for a string literal, since there was only a single usage.
Removed ✅

Reply:
Awesome! Thanks!